HAB5/hab5.tex

```diff
@@ -272,9 +272,9 @@ \subsection{Prerequisite Skills}
 Proficiency in Python (CS 1 or equivalent) is required for implementing PLANTHAB image capturing and processing software.
 Experience with Python computer vision and machine learning tools is required for developing the advanced CV objectives like object detection and segmentation. A minimum of 2-3 dedicated individuals will be required for the project.
 Electrical engineering skill set varies greatly based on method of implementation---purchasing ready-to-assemble products to design and fabrication of custom PCBs. A single eiectrical engineer could, with significant effort, design and implement the sysem. A safer approach would be 2-3 with moderate experience; further talent being used to move the project away from premade subcomponents.
-The result is that we have the opportunity to tune implementation based on the EEs we find.
-Understanding of basic circuits and connections/interfacing is still required.
-An understanding of the fundamentals of heat transfer is recommended in order to do basic thermal analysis. A team of 2-3 mechanicals would be required for the project itself. Additional talent can be used to probe the question of camera stabilization and other long-term niceties.
+The result is that we have the opportunity to tune implementation based on the EEs we find. An effective minimum of two people working towards electrical systems is desirable.
```

Comment:
If the goal is to reuse the currently available hardware, you can get away with one person. You'll be more dependent on software than hardware for this HAB. Focus on perfboard and breakout board based hardware to reduce the need for EE focused people.